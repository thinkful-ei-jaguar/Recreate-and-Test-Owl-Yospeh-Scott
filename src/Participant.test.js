import React from 'react';
import ReactDOM from 'react-dom';
import Participant from './Participant'
import renderer from 'react-test-renderer';


describe('Participant Component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Participant />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<Participant />)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });
});
