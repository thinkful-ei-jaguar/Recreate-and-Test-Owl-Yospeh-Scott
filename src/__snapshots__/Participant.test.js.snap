// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participant Component renders the UI as expected 1`] = `
<div
  className="participant"
>
  <img
    alt="avatar"
    className="avatar"
  />
  <div
    className="information"
  >
    <p />
    <p />
    <p />
  </div>
</div>
`;
